README.md

```diff
@@ -2,11 +2,42 @@
 
 A very simple calculator designed to teach software testing.
 
+
+## What does Red Green Refactor mean?
+
+**NOTE:** If you want ot just dive into code skip to the Quick Start section.
```

Comment:
To instead of ot. Also, maybe provide a relative link to the Quickstart section.